FALSEHOODS.md

```diff
@@ -235,3 +235,10 @@ surprising how many false assumptions programmers continue to make about them.
     during, or after the actual implementation of numbering plan changes in the
     real world. The actual date on which a phone number range becomes active may
     not always adhere to official announcements.
+
+1.  **Phone numbers are numbers**
+
+    Never try to store phone numbers in an int or any other kind of numeric
```

Comment:
nit: in -> as?

(also, ints are too small :) but we probably don't want to get into that.)